.github/workflows/pr.yml

```diff
@@ -5,6 +5,9 @@ on:
     - '*'
 concurrency: pr-${{ github.ref }}
 jobs:
+  quality-check:
+    name: quality-check
```

Comment:
We should use `QC` or `Quality Check` so its style is matched with the other job.